libs/platform/src/lib/form/checkbox/checkbox.component.ts

```diff
@@ -103,7 +103,7 @@ export class CheckboxComponent extends BaseInput implements AfterViewInit {
      * Emitting checked event for non-form checkbox
      */
     @Output()
-    readonly checkedChange: EventEmitter<PlatformCheckboxChange> = new EventEmitter<PlatformCheckboxChange>();
+    readonly checkedChange = new EventEmitter<any>();
```

Comment:
I'm not adding `any` here, it was always there under `payload` property, I'm just inlining it basically.

The problem is current checkbox implementation is a bit odd: normally you would have it emitting `true|false|null` as values (null for indeterminate state). But we allow to pass custom values 
```typescript
export interface FdCheckboxValues {
    trueValue?: any;
    falseValue?: any;
    thirdStateValue?: any;
}
```
Basically it means this `checkedChange` can output 3 different types. Do you think it will bring any additional value, given the fact there will be no strict types anyway since checkbox exists within template, not ts.